### PR TITLE
Heading Added

### PR DESCRIPTION
**Pull request by atharvagadkari05**

### CODE REVIEW

File: src/tests/__heading.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title></title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="../lib/eccentric.css" rel="stylesheet">
+    </head>
+    <body>
+        <h1 class="et-h1">This is a Heading</h1>
```
Comment (yashsehgal) [inline src/tests/__heading.html:10]: Nice unit tests :)

File: src/lib/eccentric.scss [hunk 2]
```diff
@@ -1,44 +1,149 @@
...
+  --et-red-primary: #E73A3A;    
+  --et-invalid: #E73A3A;  
+  --et-danger: #E73A3A;
+  --et-red-secondary: #FF8282;  
 
-  /**
-    Here all the CSS modules will be linked
-    Modules: animations.css, buttons.css, components.css, contents.css, effects.css, footers.css, headers.css, images.css
-  */
+  --et-green-primary: #00D43B;
```
Comment (yashsehgal) [inline src/lib/eccentric.scss:33]: Color UX is already updated

File: src/lib/eccentric.scss [hunk 3]
```diff
@@ -1,44 +1,149 @@
...
+  background-color: var(--et-yellow-primary);
+  padding: 0.6em;
+  border: none;
+  margin: 0;
+  font-family: var(--et-sans-serif);
+  border-radius: 4px;
+  font-size: inherit;
+  line-height: inherit;
+}
+h1,.et-h1,h2,.et-h2,h3,.et-h3,h4,.et-h4,h5,.et-h5,h6,.et-h6{
```
Comment (yashsehgal) [inline src/lib/eccentric.scss:112]: Alright, This feature-request is done, and comes under the sprint 01 #77

**Review by yashsehgal — APPROVED** [review]
LGTM!

File: src/lib/eccentric.scss (hunk 3 above)
Comment (atharvagadkari05) [reply src/lib/eccentric.scss:112]: Thanks @yashsehgal

**Pull request merged**